### PR TITLE
adding pretty html for df

### PR DESCRIPTION
**Pull request by ofrighil**

### CODE REVIEW

File: relevanceai/dataset_api/dataset.py [hunk 1]
```diff
@@ -612,6 +612,11 @@ def head(
                 )
                 return pd.json_normalize(head_documents).head(n=n)
 
+    def _repr_html_(self):
+        return pd.json_normalize(
+            self.get_documents(dataset_id=self.dataset_id)
+        )._repr_html_()
```
Comment (boba-and-beer) [inline relevanceai/dataset_api/dataset.py:618]: can this be `show_json` ? We want to ideally add support for images/audio if possible!

Comment (ofrighil) [reply relevanceai/dataset_api/dataset.py:618]: Do you mean to make it like a try/except like in `head`?

Comment (boba-and-beer) [reply relevanceai/dataset_api/dataset.py:618]: Yep - or can you just use `head` ?

Comment (ofrighil) [reply relevanceai/dataset_api/dataset.py]: Reusing `head` would not work, I resolved it in a different way.

**Review by boba-and-beer — APPROVED** [review]

**Pull request merged**